### PR TITLE
Updated tenant call blocking cmdlets

### PR DESCRIPTION
**Pull request by JensTrier**
Cleanup help files related to the tenant call blocking feature

- Added help files for *-CsInboundExemptNumberPattern. These are the cmdlets to use to configure exemptions
- Removed the help files for *-CsTenantBlockedNumberExceptionPattern, since these cmdlets have no impact on tenant call blocking
- Updated *-CsTenantBlockedCallingNumbers to reflect the new cmdlets for exemption
- Updated the TOC with the new cmdlets and remove the deleted cmdlets

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsInboundExemptNumberPattern.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
+---
+external help file: Microsoft.Rtc.Management.Hosted.dll-help.xml 
+online version: https://docs.microsoft.com/powershell/module/skype/get-csinboundexemptnumberpattern
+applicable: Skype for Business Online and Teams
+author: jenstr
```
Comment (get-itips) [inline skype/skype-ps/skype/Get-CsInboundExemptNumberPattern.md:5]:
```suggestion
author: jenstrier

```

Comment (get-itips) [reply skype/skype-ps/skype/Get-CsInboundExemptNumberPattern.md:5]: Hello Jens, _author_ metadata should be a github username.

**Comment by get-itips** [comment]
Hello Jens, how are you? hope you are doing great.

> Removed the help files for *-CsTenantBlockedNumberExceptionPattern, since these cmdlets have no impact on tenant call blocking

Are these going to be removed from the module soon? in 2.3.1 version there are still there, and, as people tend to create issues reporting missing cmdlet articles, we tend to maintain the article and add a deprecated note to it, however, if they are going to be removed soon from the code, we could also go on and remove the articles. what do you think @dariomws @tseward ?

File: skype/skype-ps/skype/Get-CsInboundExemptNumberPattern.md (hunk 1 above)
Comment (JensTrier) [reply skype/skype-ps/skype/Get-CsInboundExemptNumberPattern.md]: Thanks! I've updated the files

**Comment by JensTrier** [comment]
> Hello Jens, how are you? hope you are doing great.
> 
> > Removed the help files for *-CsTenantBlockedNumberExceptionPattern, since these cmdlets have no impact on tenant call blocking
> 
> Are these going to be removed from the module soon? in 2.3.1 version there are still there, and, as people tend to create issues reporting missing cmdlet articles, we tend to maintain the article and add a deprecated note to it, however, if they are going to be removed soon from the code, we could also go on and remove the articles. what do you think @dariomws @tseward ?

Yes, they will be removed. I'm also updating the Phone System documentation to not reference them at all.

**Comment by get-itips** [comment]
> Yes, they will be removed. I'm also updating the Phone System documentation to not reference them at all.
Ok, good to know! thanks

**Comment by JensTrier** [comment]
> > Yes, they will be removed. I'm also updating the Phone System documentation to not reference them at all.
> 
> Ok, good to know! thanks

BTW can't guarantee when they will be removed.

**Review by dariomws — APPROVED** [review]
Agree with @get-itips. If the CsTenantBlockedNumberExceptionPattern cmdlets won't be removed from the code in the short term, it's a good idea to leave them with a "deprecated" message.

Thanks!

**Comment by JensTrier** [comment]
> Agree with @get-itips. If the CsTenantBlockedNumberExceptionPattern cmdlets won't be removed from the code in the short term, it's a good idea to leave them with a "deprecated" message.
> 
> Thanks!

Thanks! So how to remove the deletes from the PR and replace them with deprecated messages?

**Comment by dariomws** [comment]
It isn't directly available on the GitHub web interface. If you are using GitHub Desktop, there is an option to revert commits.

Maybe the best option would be to create a new PR to add the files again.

**Comment by JensTrier** [comment]
> It isn't directly available on the GitHub web interface. If you are using GitHub Desktop, there is an option to revert commits.
> 
> Maybe the best option would be to create a new PR to add the files again.

I think I manage to bring them back into this PR with updated message about deprecation and links to the new cmdlets. Please check

**Comment by dariomws** [comment]
It looks good. Thanks!

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Comment by v-dihans** [comment]
Hi @JensTrier  

**Following are blocking issues that need to be resolved before we can merge:**

skype/skype-ps/skype/Set-CsInboundExemptNumberPattern.md
- Line 200: Change "http" to "https"

&nbsp;
**Following are non-blocking issues that should be resolved now or within three business days in another PR:**

All articles
- The articles have new/unknown metadata ("external help file" and "online version") that we're not used to seeing. Can you provide a link to some guidance on this?

skype/skype-ps/skype/Get-CsInboundExemptNumberPattern.md
skype/skype-ps/skype/New-CsInboundExemptNumberPattern.md
skype/skype-ps/skype/Remove-CsInboundExemptNumberPattern.md
skype/skype-ps/skype/Set-CsInboundExemptNumberPattern.md
- If it has not already been done, add a TOC entry for each of the new articles

Absolute links to Microsoft docs in articles
- We have new criteria that makes full (absolute) links to Microsoft docs articles blocking. Please make the links to Microsoft docs articles relative or site relative links.

***

**Reference**

[Criteria used for pull request reviews](https://review.docs.microsoft.com/en-us/office-authoring-guide/pr-criteria?branch=master)

**Blocking**
- All instances of "http" are changed to "https" in Microsoft links.
- Links to other files in Microsoft Docs aren’t full-path URLs:
    - For links to files in the same doc set (folder), use file-relative links (include the file name extension).  Example: `safe-attachments.md`
    - For links to files in different doc sets (folders) or repos, use site-relative links (don't include the file name extension).
    - Example: `/microsoft-365/security/defender-365-security/safe-attachments`
    - Tip:   To get the site-relative link, remove`https://` from the beginning of the full path, and remove the file name extension.
    - For more information, see [Links](https://review.docs.microsoft.com/en-us/help/contribute/links-how-to?branch=master).

**Non-blocking**
- When a new article is added, *the same PR must also contain the updated TOC* with the entry for the new article. Exceptions are allowed when the file has **NOINDEX:NO FOLLOW** in the metadata.

***

Thanks

**Comment by JensTrier** [comment]
@v-dihans Thanks. I changed http to https.

For the unknown metadata - I copied from this https://raw.githubusercontent.com/MicrosoftDocs/office-docs-powershell/master/skype/skype-ps/skype/Get-CsInboundBlockedNumberPattern.md, but I can remove them?

**Comment by v-dihans** [comment]
> @v-dihans Thanks. I changed http to https.
> 
> For the unknown metadata - I copied from this https://raw.githubusercontent.com/MicrosoftDocs/office-docs-powershell/master/skype/skype-ps/skype/Get-CsInboundBlockedNumberPattern.md, but I can remove them?

Hi @JensTrier  

Thanks for the fix.  We're looking into the metadata question so we can understand it better, but you don't need to remove it.  Add the sign-off label when you are ready to have the PR reviewed again.

Thanks

**Comment by JensTrier** [comment]
@v-dihans 

I believe the updated TOC is in this PS, but not sure about the issue with "Absolute links to Microsoft docs in articles" Where did you find this?

**Comment by v-dihans** [comment]
@JensTrier  

I don't see an updated TOC file in the PR.  Just so long as the new articles are added to a TOC, it's fine.  For future though, it's required that new articles be added to a TOC file, and that the updated TOC file be included in the same PR that has the new articles.

We have new blocking criteria that says links to Microsoft docs need to be relative or site relative. You have absolute links (https://docs.microsoft.com/[file path]) in some files, in the metadata.  Since the usage is in the metadata, we only non-blocked for it.  They should be relative or site relative links. (Example, file skype/skype-ps/skype/Get-CsInboundExemptNumberPattern.md at Line 3). 

Thanks

**Comment by JensTrier** [comment]
@v-dihans 

I thought the skype.md file was the TOC? If it's not the TOC, please let me know where I can find it and I'll update it :-)

I added them there
image

I don't seem to be able to add the required label back (or don't know how to do it :-)), so if it's ready please add it.

**Pull request merged**